File: k9mail/src/test/java/com/fsck/k9/message/html/HtmlHelper.java
package com.fsck.k9.message.html;


import org.jsoup.Jsoup;
import org.jsoup.safety.Whitelist;


public class HtmlHelper {
    public static String extractText(String html) {
        return Jsoup.clean(html, Whitelist.none());
    }
}

File: app/core/src/test/java/com/fsck/k9/message/html/HtmlHelper.kt
package com.fsck.k9.message.html

import org.jsoup.Jsoup
import org.jsoup.safety.Safelist

object HtmlHelper {
    @JvmStatic
    fun extractText(html: String): String {
        return Jsoup.clean(html, Safelist.none())
    }
}
